warc_metadata_sidecar.py

```diff
@@ -188,12 +203,15 @@ def metadata_sidecar(archive_dir, warc_file, operator=None, publisher=None):
                 continue
             if 'text/dns' in record.rec_headers.get_header('Content-Type'):
```

Comment:
With the new style of check you are doing to handle ARC DNS records, are we able to remove this 'text/dns' check?